negative.py

```diff
@@ -0,0 +1,22 @@
+# Import the dependencies
+import argparse
+import cv2 #image processing lib
+import numpy as np #matrix manipulation
+from time import sleep #for slowing down the process to make progress visible 
+from tqdm import tqdm as tqdm # for progess bar 
+
+#construct the argument parse and parse the arguments
+ap = argparse.ArgumentParser()
+ap.add_argument("-i", "--image", required=True, help="path to input image")
+args = vars(ap.parse_args())
+
+# reading the image
+img = cv2.imread((args["image"]))
+negative = 255 - img
+
+cv2.imwrite("assets/negative.jpg", negative)
```

Comment:
Can you please give an input and output image in the description of the PR as a sample

Reply:
Oh! Sorry.
 Yes, for sure will do this change